lib/galaxy/datatypes/hubAssembly.py

```diff
@@ -0,0 +1,59 @@
+"""
+HubAssembly datatype
+"""
+import logging
+import galaxy.version as version
+
+# Support for Galaxy <= 16.01
+if version.VERSION_MAJOR <= "16.01":
+    from galaxy.datatypes.images import Html
+else:
+    from galaxy.datatypes.text import Html
```

Comment:
Hrmm, I was hoping to have more feedback about this. But if you both think I don't need to support older versions of Galaxy, I will go for it! (I guess it makes sense if it is shipped with 16.07...but could be cherry-picked :/).